templates/adp-backend-template-node/template.yaml

```diff
@@ -316,6 +316,7 @@ spec:
             - AZR-ADP-DEV1
             - AZR-ADP-TST1
             - AZR-ADP-PRE1
+            - AZR-ADP-PRD1
```

Comment:
is the associated configuration in the ADO Projects and the Portal API to ensure all Agents pools and envs are created? @rajeshkaremane  @ppotturi @ajaysoni-cap @kbabigumira